index.html

```diff
@@ -95,7 +95,7 @@ <h1>Multi-Post <br>Stories</h1>
             <li><h5>css</h5></li>
             <li><h5>javascript</h5></li>
          </ul>
-         <button class="btn">See Project</button>
+         <button type="button">See Project</button>
         </div>
     
     <div class="about">
```

Comment:
- [ ] The about section should be aligned with flexbox so it matches the design.
![image](https://user-images.githubusercontent.com/74744787/185768108-960503c9-3974-4fd1-aa7c-5e8ec27d1166.png)